ewc-rfc005-issue-legal-person-identification-data.md

```diff
@@ -31,34 +31,42 @@ The LPID issuance process is implemented by PID providers according to national
 
 The second approach has no dependency to the implementaion of the wallet or the wallet solution of the User. An end user representing the legal person goes directly to the eService of the PID Provider to apply for/request an LPID. After completeing the eService procedure, the PID Provider initiates a offer response to the wallet instance of the legal person which endpoint the representative of the lagal person supplied. The endpoint can be a url or a DID for instance. All communication between wallets is done in the backend.
 
-The picture below shows a generic outline of the LPID issuance process, the actual implemntation depends on national regulations.
-![Generic LPID issuance process](images/Generic%20LPID%20issuance%20process.png)
+The third approach combine the wallet and the E-service into one service. Where the user has a seemless experience and there is no need for the user to switch between webservices. The switching between the wallet and the e-service is done in through machine to machine in the background. 
+
+The picture below shows a generic outline of the LPID issuance process, the actual implemntation depends on national regulations and the order of the action does not need to be the same. 
+![Screenshot 2024-06-06 at 14 49 14](https://github.com/EWC-consortium/eudi-wallet-rfcs/assets/104071742/a0f16ef4-b047-496f-bf3c-5e32b1638b35)
+
 
 *Figure 1. Generic LPID issuance process*
 
-**Process steps:**
-Start: Natural person visists PID providers eService for LPID issuance
-1. The natural person authenticates using LoA High (any LoA High certified method). (Not done during pilot as there is no LoA High available yet.)
-2. PID Provider verifies the natural person credentials.
-3. The natural person signs and submits the application for an LPID. (signing using existing methods)
-4. PID Provider verifies the signature of the natural person.
-5. PID Provider verifies the powers of the natural person to represent the legal person for which the LPID is requested (according to assumptions as national regulation isn’t in place yet).
-6. PID Provider verifies the status of the legal person (according to assumptions as national regulation isn’t in place yet).
-7. Verification of results from steps 4-6.
-8. PID Provider asks the natural person for a wallet id or endpoint for the wallet of the legal person. (Endpoint in first phase to avoid DID registration)
-9. The natural person submits wallet id/endpoint. 
-10. PID Provider creates the schema with data for the LPID.
-11. PID Provider publishes revocation information for the LPID to be issued.
-12. PID Provider issues the LPID to the wallet instance of the legal person appointed by the natural person. (done by the PID Provider wallet, details shown in the following chapter.
-End: LPID is stored in wallet instance
+**Generic High level Process steps**
```

Comment:
Three actors are not enough to detail the sequence diagram for the issuance of LPID, and the sequence described below will not work as there is no LPID of the legal person yest. There is no way to authenticate the legal person at this stage. When an LPID is issued, other attestations can be requested by authentication the legal person.